guides/common/modules/ref_host-parameter-hierarchy.adoc

```diff
@@ -0,0 +1,20 @@
+[id="Host_Parameter_Hierarchy_{context}"]
+= Host Parameter Hierarchy
+
+You can access host parameters when provisioning hosts.
+Hosts inherit their parameters from the following locations, in order of increasing precedence:
+
+[cols="50%,50%",options="header"]
+|====
+| Parameter Level | Set in {ProjectWebUI}
+| *Globally defined parameter* | *Configure* > *Global parameters*
+| *Organization-level parameters* | *Administer* > *Organizations*
+| *Location-level parameters* | *Administer* > *Locations*
+| *Domain-level parameters* | *Infrastructure* > *Domains*
+| *Subnet-level parameters* | *Infrastructure* > *Subnets*
+| *Operating system-level parameters* | *Hosts* > *Operating Systems*
+| *Host group-level parameters* | *Configure* > *Host Groups*
+| *Host parameters* | *Hosts* > *All Hosts*
+|====
+
+Note that globally defined parameters apply to all hosts.
```

Comment:
I don't know whether this sentence is even necessary. Otherwise they wouldn't be called global.

Reply:
Removed.